setuptools/build_meta.py

```diff
@@ -289,7 +297,7 @@ def _arbitrary_args(self, config_settings: _ConfigSettings) -> Iterator[str]:
 
 
 class _BuildMetaBackend(_ConfigSettingsTranslator):
-    def _get_build_requires(self, config_settings, requirements):
+    def _get_build_requires(self, config_settings: _ConfigSettings, requirements):
```

Comment:
Doesn't cause any issue. I didn't add annotations to `_get_build_requires` because it's not typed in typeshed. `config_settings` param being typed is incidental to adding all `_ConfigSettings`.

But since you're mentioning it here and it's not an issue, I've added the type annotation to `requirements` param